pkg/reconcilermanager/controllers/volumes.go

```diff
@@ -32,14 +32,14 @@ const GitCredentialVolume = "git-creds"
 // HelmCredentialVolume is the volume name of the git credentials.
 const HelmCredentialVolume = "helm-creds"
 
-// PrivateCertVolume is the volume name of the private certificate.
-const PrivateCertVolume = "private-cert"
+// CACertVolume is the volume name of the CA certificate.
+const CACertVolume = "ca-cert"
 
-// PrivateCertKey is the secret key that contains the private certificate.
-const PrivateCertKey = "cert"
+// CACertKey is the secret key that contains the CA certificate.
+const CACertKey = "cert"
```

Comment:
This name and description is hella confusing, but I can't think of one that's less confusing... SecretKeyCACert? It's the name of a key in the Secret's data map whose calue holds the CA public cert as a string, but a "CertKey" makes it sound like a private key or something.

Reply:
Maybe `CACertSecretKey`?

Reply:
sure